grabbit/src/main/groovy/com/twcable/grabbit/client/batch/steps/http/CreateHttpConnectionTasklet.groovy

```diff
@@ -41,11 +45,19 @@ import org.springframework.batch.repeat.RepeatStatus
 /**
  * This tasklet opens an HTTP session for the current job, and places the input stream on the {@link ClientBatchJobContext}
  */
+@CompileStatic
+@Slf4j
 class CreateHttpConnectionTasklet implements Tasklet {
 
     @Override
     RepeatStatus execute(StepContribution contribution, ChunkContext chunkContext) throws Exception {
-        HttpResponse response = doRequest(chunkContext.stepContext.jobParameters)
+        final Map jobParameters = chunkContext.stepContext.jobParameters
+        HttpResponse response = doRequest(jobParameters)
+        final int statusCode = response.getStatusLine().statusCode
+        if(statusCode != SC_OK) {
+            log.warn "Received a status of ${statusCode} when attempting to create a connection to ${jobParameters.get(ClientBatchJob.HOST)}. Bailing out.."
```

Comment:
Trivial: Is that last sentence supposed to be a statement or unfinished thought? (i.e., It should end either in a period or an ellipsis, but not two dots.)